pkg/virt-handler/vm.go

```diff
@@ -687,6 +690,31 @@ func (d *VirtualMachineController) migrationSourceUpdateVMIStatus(origVMI *v1.Vi
 	return nil
 }
 
+type NetworkStatus struct {
+	Name      string   `yaml:"name"`
+	Ips       []string `yaml:"ips"`
+	Interface string   `yaml:"interface"`
+}
+
+// Using the downward API, look for dedicated migration network net1 and, if found, set migration IP to its IP
+func findMigrationIP(vmi *v1.VirtualMachineInstance, migrationIp *string) {
+	var networkStatus []NetworkStatus
+
+	dat, err := ioutil.ReadFile("/etc/podinfo/network-status")
+	if err != nil {
+		return
+	}
+	err = yaml.Unmarshal(dat, &networkStatus)
+	if err != nil {
+		return
+	}
+	for _, ns := range networkStatus {
+		if ns.Interface == "net1" && len(ns.Ips) > 0 {
```

Comment:
As suggested by you in another comment, here it would be great if you would use the multus network name and extract all necessary info from there. Only this way it does not matter if we have one or more multus netwoks. Otherwise any kind of preset mechanism which would add multus annotations could cause trouble.